bundles/aws-bundle/src/main/resources/META-INF/services/org.apache.gravitino.credential.CredentialProvider

```diff
@@ -0,0 +1,20 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+org.apache.gravitino.s3.credential.S3TokenProvider
```

Comment:
This is AWS Hadoop Credential provider names
```
    org.apache.hadoop.fs.s3a.TemporaryAWSCredentialsProvider,
    org.apache.hadoop.fs.s3a.SimpleAWSCredentialsProvider,
    software.amazon.awssdk.auth.credentials.EnvironmentVariableCredentialsProvider,
    org.apache.hadoop.fs.s3a.auth.IAMInstanceCredentialsProvider
```
Do we need to refer to them? Many people may be used to using them.

Reply:
This is for developers only, the user only care about credential provider type, like `s3-token` for this,   I'm ok to use this name, do you have some suggestion?

Reply:
It's ok if it's only used by developers. But we need to make sure that `s3-token` is easy to understand by users.

Reply:
Do you think `s3-token` and `s3-secret-key` is clear for users?

Reply:
What name do other system use?

Reply:
polaris use `storage-type` = `s3` please refer to  https://polaris.io/#tag/Polaris-Entities/Catalog,   uc get s3 properties if the location is s3 schema, see https://docs.unitycatalog.io/server/configuration/

Reply:
Two consideration points.
1. GCP is `gcp-token`? Should be this `aws-token`?
2. I am not sure token is correct enough? Does S3 have multiple tokens? Do we support all?

Reply:
1. GCP is `gcs-token`,  I prefer to keep `s3-token`, because it's used for s3 storage. 
2. We just support token from AsummRole, I think the name is simple and clear.

Reply:
OK. I just discuss some possible names to make sure. I'm ok with current name.